hdf/src/hconv.h

```diff
@@ -57,8 +57,8 @@
 /*****************************************************************************/
 /* CONSTANT DEFINITIONS                                                      */
 /*****************************************************************************/
-/* Generally Big-Endian machines */
-#if !defined(INTEL86) && !defined(MIPSEL) && !defined(DEC_ALPHA) && !defined(I860) && !defined(SUN386) && !(defined(__ia64) && !(defined(hpux) || defined(__hpux))) && !defined(__x86_64__)
+/* Big-Endian machines */
+#ifdef __BIG_ENDIAN__
```

Comment:
This fixed my arm config of my bazel based run of hdf4.  Thanks!

Reply:
Great, thanks!